pynxtools_apm/utils/load_ranging.py

```diff
@@ -233,7 +233,7 @@ def extract_data_from_rng_file(file_path: str, template: dict, entry_id: int) ->
 def extract_data_from_rrng_file(file_path: str, template: dict, entry_id) -> dict:
     """Add those required information which an RRNG file has."""
     print(f"Extracting data from RRNG file: {file_path}")
-    rangefile = ReadRrngFileFormat(file_path)
+    rangefile = ReadRrngFileFormat(file_path, unique=False)
```

Comment:
Not really necessary, because `unique` is the default. But fine if you want to have it for explicitness.

Reply:
Exactly